certified-connectors/Redque/Readme.md

```diff
@@ -0,0 +1,58 @@
+# Redque
+Redque provides a powerful REST API.  Using this API, you can manage your documents, accounts, folders, enums and configurations in an Redque system.  Very often, you may want to leverage those functions in your accounting application or in your process automation.
+
+## Publisher: Redque s.r.o.
+
+## Prerequisites
+You will need the following to proceed:
+* A Redque account (register here: https://register.redque.com/)
+* A Redque subscription
+
+## Supported Operations
+The connector supports the following operations:
+
+### Configuration
+* `Return document to issuer config`: Retrieve current configuration for return to sender action
+* `Update template`: Update template for return o issuer function
+### Documents
+* `Get image`: Gets link to an image of a given page
+* `Return to issuer`: Returns document to the issuer for corrections
+* `List documents`: List filtered documents based on parameters
+* `Export document`: Export document to accounting provider
+* `Donwload document`: Downloads document's file or gets URL
+* `Get document's metadata`: Gets document's metadata
+* `Update document`: Updates document
+* `Lock document`: Locks (opens) document preventing other users from modifying it
+* `Unlock document`: Unlocks locked (closes) document to allow other users to modify its contents
+* `Grant access`: Grants document access to specified user
```

Comment:
@jirisodomka all files should be free of grammatical errors and typos. please change "acces" to "access". You have the same typos in your readme and swagger file as well please fix both files.